content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
+weight: 910
+description: Getting started with etcd on Linux or macOS
```

Comment:
How about this description: _Get etcd up and running in less than 5 minutes!_

Reply:
I like it!